scripts/qa_crowbarsetup.sh

```diff
@@ -1350,6 +1350,22 @@ function custom_configuration()
             if [ -n "$networkingplugin" ] ; then
                 proposal_set_value neutron default "['attributes']['neutron']['networking_plugin']" "'$networkingplugin'"
             fi
+            # For Cloud5+, proposal attribute names changed
+            if iscloudver 5plus; then
```

Comment:
Argh, no, that's wrong. Hrm, it's really: "if Cloud 5 and later than M4, or Cloud 6+, then"